packages/framer-motion/src/motion/__tests__/ssr.test.tsx

```diff
@@ -146,7 +146,7 @@ function runTests(render: (components: any) => string) {
         )
 
         expect(div).toBe(
-            '<div style="transform:translateX(100px) translateY(200px)"></div>'
```

Comment:
Likewise we definitely don't want to set this if it wasn't explicitly set by the user - unless this is an SVG with a `transform`